myapp/tests/test_jobs.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from ..scheduler.stock_api_update import stock_api_update
+from ..models import Stock
+
+
+class test_api(TestCase):
+
+    def test_api_update(self):
+        stock_api_update()
+        value_1 = Stock.objects.filter(top_rank=1)[:1][0].last_modified
+        stock_api_update()
+        value_2 = Stock.objects.filter(top_rank=1)[:1][0].last_modified
+        self.assertNotEqual(value_1, value_2)
```

Comment:
Yeah, this is what I was talking about in the meeting yesterday, nice!  Maybe you could use `assertAlmostEqual()` to check if the time difference is approximately 10 seconds as opposed to just checking if the times are not the same?

Reply:
in this section i'm testing the API not the job , so the difference is so little second or less